pytest_mpl/plugin.py

```diff
@@ -131,7 +144,7 @@ def item_function_wrapper(*args, **kwargs):
 
             baseline_remote = baseline_dir.startswith('http')
 
-            with plt.style.context(style):
+            with plt.style.context(style), switch_backend(backend):
```

Comment:
@dopplershift looks like this is causing issues with Python 2.6. Does Matplotlib still support Python 2.6?